addon/components/pikaday-input.js

```diff
@@ -40,5 +41,25 @@ export default Ember.Component.extend({
 
   setDate: function() {
     this.get('pikaday').setDate(this.get('value'), true);
-  }.observes('value')
+  }.observes('value'),
+
+  determineYearRange: function() {
+    var yearRange = this.get('yearRange');
+
+    if (yearRange) {
+      if (yearRange.indexOf(',') > -1) {
+        var yearArray =  yearRange.split(',');
+
+        if (parseInt(yearArray[1]) === 0) {
```

Comment:
When using `parseInt` we want to make sure that we use base ten by passing `10` as second argument: `parseInt(yearArray[1], 10)`